### ISSUE TITLE
still maintained?

### ISSUE THREAD

**Issue by smcclstocks** [schematics/schematics#332]
I noticed the initial author is no longer working on this & quite a few pull requests are sitting. Any other maintainers still working on this? Thanks.

**Comment by kracekumar** [comment]
I merge PR and release when needed.

On Thu, Oct 8, 2015 at 5:56 AM, smccl stocks <notifications@github.com>
wrote:

> I noticed the initial author is no longer working on this & quite a few
> pull requests are sitting. Any other maintainers still working on this?
> Thanks.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/schematics/schematics/issues/332>.
>



-- 
Regards
Kracekumar Ramaraj
http://kracekumar.com

**Comment by smcclstocks** [comment]
thanks.

**Comment by jmsdnns** [comment]
I see that the issue is closed. It is true that I (formerly j2labs) am no longer working on it but I still watch what's happening. I passed the torch to a group of maintainers who were already committing code and they have done a superb job up to now.

Personal trivia, I recently started working with Python again. Maybe I'll do some more work on it if the need comes up, but I must say... it mostly does what I need already. :)
